.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
+- '10.21.0'
 - '8.8.1'
```

Comment:
@Pholey node 6 got EOL'd.......... figured it isn't worth it trying to wrangle deps anyway

Reply:
@AnEmortalKid sounds good to me